src/os/filter/impl/ecql/areaformatter.js

```diff
@@ -49,6 +83,12 @@ os.filter.impl.ecql.AreaFormatter.prototype.format = function(feature) {
   if (geom) {
     geom = geom.clone().toLonLat();
     os.geo.normalizeGeometryCoordinates(geom);
+
+    if (!this.supportsAltitude) {
+      var poly = /** @type {ol.geom.MultiPolygon|ol.geom.Polygon} */ (geom);
+      poly.setCoordinates(os.filter.impl.ecql.AreaFormatter.stripAltitude(poly.getCoordinates()));
```

Comment:
`os.filter.impl.ecql.AreaFormatter.stripAltitude` doesn't return anything, so this sets the coordinates to undefined.